src/components/ErrorsNotify.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
Maybe 'MessageContainer.vue' or 'MessageComponent.vue' would be a better name for this component?

Reply:
Changed.